bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -65,6 +65,11 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                0,
                "number of consensus operations that can be executed in parallel "
                "1 <= concurrencyLevel <= 30");
+  CONFIG_PARAM(numWorkerThreadsForBlockIO,
```

Comment:
Not necessary to add here - could be done by a dynamic configuration mechanism

Reply:
Other parameters in this layer taken from replica config and for now I don't expect we will "play" with this parameter much, else than the deault. So I'll leave it like that.